tests/unit/test_specs.py

```diff
@@ -162,3 +181,25 @@ def test_validate_supported_bundles_fail():
         specs.validate_supported_bundles(TEST_ITEM_TYPE,
                                          'analytic',
                                          ALL_PRODUCT_BUNDLES)
+
+
+def test_get_supported_assets_success():
+    supported_assets = specs.get_supported_assets(TEST_ITEM_TYPE)
+    supported_assets.sort()
+    ALL_SUPPORTED_ASSET_TYPES_PSSCENE.sort()
+    assert supported_assets == ALL_SUPPORTED_ASSET_TYPES_PSSCENE
+
+
+def test_get_supported_assets_not_supported_item_type():
+    with pytest.raises(specs.SpecificationException):
+        specs.get_supported_assets('notsupported')
+
+
+def test_validate_asset_type():
```

Comment:
I'm not sure what this is testing. Could you clarify in a docstring? Also, could you add a postfix to the test name that allows it to be run independent of `test_validate_asset_type_notsupported` and is related to the test purpose?